src/CHANGES.md

```diff
@@ -130,7 +130,7 @@
 -   \[FIX] Mix up (typo) between ficiduals and landmarks in EEG spec [#754](https://github.com/bids-standard/bids-specification/pull/754) ([rob-luke](https://github.com/rob-luke))
 -   \[INFRA] updating remark, CIs, contributor docs [#745](https://github.com/bids-standard/bids-specification/pull/745) ([sappelhoff](https://github.com/sappelhoff))
 -   \[FIX] schema for i/eeg coordsys+elecs: sub-ses-acq-space are allowed entities [#743](https://github.com/bids-standard/bids-specification/pull/743) ([sappelhoff](https://github.com/sappelhoff))
--   \[DOC] move schema documentation into the schema folder [#740](https://github.com/bids-standard/bids-specification/pull/740) ([Remi-Gau](https://github.com/Remi-Gau))
+-   \[DOC] move schema documentation into the schema directory [#740](https://github.com/bids-standard/bids-specification/pull/740) ([Remi-Gau](https://github.com/Remi-Gau))
```

Comment:
Auto-generated CHANGES will just get reverted. We should exclude this from the PR.

Reply:
I just renamed those PR titles instead.